src/_locales/en.config

```diff
@@ -174,10 +174,10 @@ Dev tools
 #==== Store listing ====
 
 @store_listing
-This eye-care extension enables night mode creating dark themes for websites on the fly. Dark Reader inverts bright colors making them high contrast and easy to read at night.
+This eye-care extension enables night mode by creating dark themes for websites on the fly. Dark Reader inverts bright colors, making them high contrast and easy to read at night.
 
-You can adjust brightness, contrast, sepia filter, dark mode, font settings and ignore-list.
+You can adjust the brightness, contrast, sepia filter, dark mode, font settings and the ignore-list.
 
-Dark Reader doesn't show ads and doesn't send user's data anywhere. It is fully open-source https://github.com/darkreader/darkreader
+Dark Reader doesn't show ads and doesn't send users' data anywhere. It's fully open-source: https://github.com/darkreader/darkreader
```

Comment:
`users'`
Is this a English thing?

Reply:
Yes, it's an English thing. In that sentence, it sounds better if the plural form of `user` is used, so we'd get `users's`. That's not fully right tho, since it ends with an `s`. Then, the `s` of `'s` is left out, so we end up with `users'`.

Reply:
Ah got it, thanks for the explaination.